### PR TITLE
feat(Video + Markdown): update react player + put videos in markdown

### PR DESCRIPTION
**Pull request by dreamwasp**
### Overview

Updates the react-player package to the newest version + makes the video component the default render component for iframed Vimeos + Youtubes

### PR Checklist

- [x] Related to JIRA ticket: [gm-890]
- [x] I have run this code to verify it works
- [x] This PR includes unit tests for the code change
- [x] This PR includes testing instructions tests for the code change
- [x] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions
* check out the Video story + the iframe sections of Markdown 
* You should be able to watch the videos    
* Profit

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](http://www.google.fr/ 'Named link title')  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/Codecademy/pull/38631)  | [Portal Env](http://www.google.fr/ 'Named link title')   |

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@57.2.3-alpha.1f6671.0<br/>@codecademy/gamut-kit@0.6.444-alpha.1f6671.0<br/>@codecademy/styleguide@68.0.5-alpha.1f6671.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@57.3.1-alpha.ef28c2.0<br/>@codecademy/gamut-kit@0.6.445-alpha.ef28c2.0<br/>@codecademy/styleguide@68.1.1-alpha.ef28c2.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://66fd9b28f914aa727f72ec84--gamut-preview.netlify.app>https://66fd9b28f914aa727f72ec84--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/66fd9b28f914aa727f72ec84>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@57.3.1-alpha.f51cd9.0<br/>@codecademy/gamut-kit@0.6.445-alpha.f51cd9.0<br/>@codecademy/styleguide@68.1.1-alpha.f51cd9.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://66fdbb35d9202c94f6db05eb--gamut-preview.netlify.app>https://66fdbb35d9202c94f6db05eb--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/66fdbb35d9202c94f6db05eb>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@57.3.1-alpha.014413.0<br/>@codecademy/gamut-kit@0.6.445-alpha.014413.0<br/>@codecademy/styleguide@68.1.1-alpha.014413.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://66fff8466d7c813ff945a53e--gamut-preview.netlify.app>https://66fff8466d7c813ff945a53e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/66fff8466d7c813ff945a53e>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@57.3.1-alpha.e82ce9.0<br/>@codecademy/gamut-kit@0.6.445-alpha.e82ce9.0<br/>@codecademy/styleguide@68.1.1-alpha.e82ce9.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://670001dd93a3915362a2c019--gamut-preview.netlify.app>https://670001dd93a3915362a2c019--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/670001dd93a3915362a2c019>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@57.3.2-alpha.7847e3.0<br/>@codecademy/gamut-kit@0.6.446-alpha.7847e3.0<br/>@codecademy/styleguide@68.1.2-alpha.7847e3.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://67004f6c4c7e411bd02c9d3c--gamut-preview.netlify.app>https://67004f6c4c7e411bd02c9d3c--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/67004f6c4c7e411bd02c9d3c>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@57.3.2-alpha.2cac30.0<br/>@codecademy/gamut-kit@0.6.446-alpha.2cac30.0<br/>@codecademy/styleguide@68.1.2-alpha.2cac30.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://670059acb153101f913f5ccd--gamut-preview.netlify.app>https://670059acb153101f913f5ccd--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/670059acb153101f913f5ccd>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@57.3.3-alpha.685447.0<br/>@codecademy/gamut-kit@0.6.447-alpha.685447.0<br/>@codecademy/styleguide@68.1.3-alpha.685447.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@57.3.3-alpha.b4412b.0<br/>@codecademy/gamut-kit@0.6.447-alpha.b4412b.0<br/>@codecademy/styleguide@68.1.3-alpha.b4412b.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6704254ffe121a0930962681--gamut-preview.netlify.app>https://6704254ffe121a0930962681--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6704254ffe121a0930962681>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@57.3.3-alpha.dd5af3.0<br/>@codecademy/gamut-kit@0.6.447-alpha.dd5af3.0<br/>@codecademy/styleguide@68.1.3-alpha.dd5af3.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://67044872db196b3a704dc43e--gamut-preview.netlify.app>https://67044872db196b3a704dc43e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/67044872db196b3a704dc43e>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@57.3.3-alpha.1532c0.0<br/>@codecademy/gamut-kit@0.6.447-alpha.1532c0.0<br/>@codecademy/styleguide@68.1.3-alpha.1532c0.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://67044e62af98783763589e54--gamut-preview.netlify.app>https://67044e62af98783763589e54--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/67044e62af98783763589e54>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@57.3.4-alpha.6a179d.0<br/>@codecademy/gamut-kit@0.6.448-alpha.6a179d.0<br/>@codecademy/styleguide@68.1.4-alpha.6a179d.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6706a7de22cb4b870a86e5ba--gamut-preview.netlify.app>https://6706a7de22cb4b870a86e5ba--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6706a7de22cb4b870a86e5ba>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@57.3.4-alpha.6a179d.0<br/>@codecademy/gamut-kit@0.6.448-alpha.6a179d.0<br/>@codecademy/styleguide@68.1.4-alpha.6a179d.0<br/></pre>

**Review by aresnik11 — COMMENTED** [review]
2 things I noticed:
1. On the Video story the props table isn't loading 
```
Invalid name '', must include alphanumeric characters 
[Read the docs](http://storybook.js.org/docs/)
```
2. the video story with a placeholder image doesn't play the video when you click, you have to click twice

File: packages/styleguide/stories/Molecules/Video/index.stories.mdx [hunk 1]
```diff
@@ -24,14 +24,14 @@ given a video URL`,
...
 <Canvas>
-  <Story name="Video">
+  <Story name="Youtube Video">
     {(args) => {
       return <Video {...args} />;
     }}
   </Story>
 </Canvas>
 
-## Here is an example with a Viemo URL
```
Comment (aresnik11) [inline packages/styleguide/stories/Molecules/Video/index.stories.mdx:34]: Viemo lol

File: packages/styleguide/stories/Organisms/Markdown/markdown-example.md [hunk 2]
```diff
@@ -347,4 +347,18 @@ Use the `printf()` function.
 
 ### Iframes
 
-<iframe src="https://player.vimeo.com/video/145702525?byline=0&portrait=0&badge=0" width="640" height="360" frameborder="0" allow="autoplay; fullscreen" allowfullscreen></iframe>
+Vimeo and Youtube video iframes will be rendered by our Video component, otherwise they'll render as stated.
```
Comment (aresnik11) [inline packages/styleguide/stories/Organisms/Markdown/markdown-example.md:350]: nit: what does render as stated mean?

File: packages/styleguide/stories/Molecules/Video/index.stories.mdx (hunk 1 above)
Comment (dreamwasp) [reply packages/styleguide/stories/Molecules/Video/index.stories.mdx:34]: lol i'll fix

File: packages/styleguide/stories/Organisms/Markdown/markdown-example.md (hunk 2 above)
Comment (dreamwasp) [reply packages/styleguide/stories/Organisms/Markdown/markdown-example.md:350]: i meant just renders the original code, i'll rephrase

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@57.3.4-alpha.01e813.0<br/>@codecademy/gamut-kit@0.6.448-alpha.01e813.0<br/>@codecademy/styleguide@68.1.4-alpha.01e813.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://670d238ac1ee79f243a3ce15--gamut-preview.netlify.app>https://670d238ac1ee79f243a3ce15--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/670d238ac1ee79f243a3ce15>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@57.3.4-alpha.76db61.0<br/>@codecademy/gamut-kit@0.6.448-alpha.76db61.0<br/>@codecademy/styleguide@68.1.4-alpha.76db61.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://670d3c72c4b31d0b2277af4f--gamut-preview.netlify.app>https://670d3c72c4b31d0b2277af4f--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/670d3c72c4b31d0b2277af4f>Deploy Logs</a></p>

**Pull request merged**